lib/app.js

```diff
@@ -642,15 +649,23 @@ module.exports = {
                                 'permission to use this Firebase');
           process.exit(1);
         }
+
         for (var i in defaultSettings) {
-          if (defaultSettings.hasOwnProperty(i) &&
+          if (i === 'public') {
+            if (settings.hasOwnProperty('public')) {
```

Comment:
We should `&&` these two `if()` statements together so they can share the same `else`